src/openrct2-ui/windows/Guest.cpp

```diff
@@ -2067,6 +2067,12 @@ void window_guest_debug_paint(rct_window* w, rct_drawpixelinfo* dpi)
     auto peep = GET_PEEP(w->number);
     auto x = w->x + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].left + 4;
     auto y = w->y + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].top + 4;
+    {
+        //args only holds one thing, so no array
+        int32_t args = {peep->sprite_index};
```

Comment:
Still not correct, but you really should initialise an integer like:

```suggestion
        int32_t args = peep->sprite_index;
```

Reply:
oops, stupid mistake. thanks for pointing that out